src/sdg/js/components/ckeditor.js

```diff
@@ -26,7 +30,10 @@ ClassicEditor.builtinPlugins = [
     LinkPlugin,
     ListPlugin,
     ParagraphPlugin,
-    Table
+    Table,
+    TableToolbar,
+    TableProperties,
+    TableCellProperties,
```

Comment:
We should not allow all sorts of properties.

Markdown only supports tables WITH a header-column but NOT with much else. So merging cells, column-titles, etc. are not allowed.